src/repl/scala/tools/nsc/interpreter/package.scala

```diff
@@ -33,186 +23,15 @@ import scala.util.Try
  *  InteractiveReader contains { history: History, completion: Completion }
  *  IMain contains { global: Global }
  */
-package object interpreter extends ReplConfig with ReplStrings {
-  type JFile          = java.io.File
-  type JClass         = java.lang.Class[_]
-  type JList[T]       = java.util.List[T]
-  type JCollection[T] = java.util.Collection[T]
-  type JPrintWriter   = java.io.PrintWriter
-  type InputStream    = java.io.InputStream
-  type OutputStream   = java.io.OutputStream
-
-  val IR = Results
-
-  implicit def postfixOps = scala.language.postfixOps // make all postfix ops in this package compile without warning
-
-  private[interpreter] implicit def javaCharSeqCollectionToScala(xs: JCollection[_ <: CharSequence]): List[String] = {
+package object interpreter {
+  private[interpreter] implicit def javaCharSeqCollectionToScala(xs: java.util.Collection[_ <: CharSequence]): List[String] = {
     import scala.collection.JavaConverters._
     xs.asScala.toList map ("" + _)
   }
 
-  private[nsc] implicit def enrichClass[T](clazz: Class[T]) = new RichClass[T](clazz)
-  private[nsc] implicit def enrichAnyRefWithTap[T](x: T) = new TapMaker(x)
-  private[nsc] def debugging[T](msg: String)(x: T) = x.tapDebug(msg)
-
-  private val ourClassloader = getClass.getClassLoader
-
-  def staticTypeTag[T: ClassTag]: ru.TypeTag[T] = ru.TypeTag[T](
-    ru.runtimeMirror(ourClassloader),
-    new TypeCreator {
-      def apply[U <: ApiUniverse with Singleton](m: Mirror[U]): U # Type =
-        m.staticClass(classTag[T].runtimeClass.getName).toTypeConstructor.asInstanceOf[U # Type]
-  })
-
-  /** This class serves to trick the compiler into treating a var
-   *  (intp, in ILoop) as a stable identifier.
```

Comment:
sadly, for now, the rabbit just moved to a different hat (will domesticate it further, but probably not in this PR)